core/src/main/java/org/apache/iceberg/rest/responses/ListTablesResponse.java

```diff
@@ -30,13 +30,15 @@
 public class ListTablesResponse implements RESTResponse {
```

Comment:
please also add some tests to `TestListTablesResponse` where the `nextPageToken` is set to `null` / `-1` / ...

Reply:
will add